examples/modbus/MBus_WTH_CO2_LCD_ETH/dds_reader/CMakeLists.txt

```diff
@@ -13,24 +13,13 @@
 #                                                                             #
 ###############################################################################
 
-include(${RESOURCE_DIR}/cmake/dds_functions.cmake)
-include(${RESOURCE_DIR}/cmake/common_functions.cmake)
-
-# Copy the USER_QOS_PROFILES before generating the code
-# List of the files to copy
-set(CO2_LCD_ETH_EXAMPLE_READER_QOS_FILE
-    "USER_QOS_PROFILES.xml"
-)
-
-copy_files(
-    CUSTOM_TARGET_NAME CO2_LCD_ETH_EXAMPLE_READER
-    OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}"
-    LIST_OF_FILES_RELATIVE_PATH ${CO2_LCD_ETH_EXAMPLE_READER_QOS_FILE})
+include("${RESOURCE_DIR}/cmake/DdsFunctions.cmake")
+include("${RESOURCE_DIR}/cmake/CommonFunctions.cmake")
 
 set(IDL_NAME "MBus_WTH_CO2_LCD_ETH_INPUT")
```

Comment:
No need to cache value to a variable, just pass the value to `rtigw_connextdds_run_codegen()` directly
```suggestion
```

Reply:
we use it to identify the subscriber name:
```
add_executable(MBus_WTH_CO2_LCD_ETH_INPUT
    "${CMAKE_CURRENT_SOURCE_DIR}/${IDL_NAME}_subscriber.c"
    ${GENERATED_SRC_FILES}
)
```

Reply:
Oh, I missed that bit. If the variable is used in more than one place, then I'd be fine with keeping it, although as you say, we can also just use the IDL name directly since it most likely won't change in the future. I'll leave it up to your preference.

Reply:
One more thing, we use the ${IDL_NAME} internally, so I've kept the IDL_NAME variable till we integrate the cmake-common repo

Reply:
What do you mean by "internally"? Do you mean we use it in the rest of `examples/modbus/MBus_WTH_CO2_LCD_ETH/dds_reader/CMakeLists.txt ` or somewhere else?

Anyway, we can keep it as is for now.

Reply:
in the `connextdds_rtiddsgen_run` (I have just renamed this function)

Reply:
I see. Somehow I missed that, and I added a comment to get rid of it :)